src/mvs_eland/E1_process_results.py

```diff
@@ -254,75 +260,155 @@ def get_results(settings, bus_data, dict_asset):
+        )
+
+        # Get flow information
+        get_flow(
+            settings=settings,
+            bus=bus_data[bus_name],
+            dict_asset=dict_asset,
+            flow_tuple=flow_tuple,
+        )
+        # Get capacity information
+        get_optimal_cap(bus_data[bus_name], dict_asset, flow_tuple)
+
+    else:
+        # Asset is connected to multiple busses, evaluate all
+        for bus_instance in bus_name:
+            flow_tuple = get_touple_for_oemof_results(
+                dict_asset[LABEL], asset_group, bus_instance
+            )
+            # Get flow information
             get_flow(
-                settings,
-                bus_data[output_name],
-                dict_asset,
-                output_name,
-                direction="output",
+                settings=settings,
+                bus=bus_data[bus_instance],
+                dict_asset=dict_asset,
+                flow_tuple=flow_tuple,
             )
-        else:
-            for bus in output_name:
-                get_flow(settings, bus_data[bus], dict_asset, bus, direction="output")
+            # Get capacity information
+            get_optimal_cap(bus_data[bus_instance], dict_asset, flow_tuple)
 
-    # definie capacities. Check if the component has multiple input or output busses
-    if OUTPUT_BUS_NAME in dict_asset and INPUT_BUS_NAME in dict_asset:
-        if not isinstance(output_name, list):
-            get_optimal_cap(bus_data[output_name], dict_asset, output_name, "output")
-        else:
-            for bus in output_name:
-                get_optimal_cap(bus_data[bus], dict_asset, bus, "output")
+    return
 
-    elif INPUT_BUS_NAME in dict_asset:
-        if not isinstance(input_name, list):
-            get_optimal_cap(bus_data[input_name], dict_asset, input_name, "input")
-        else:
-            for bus in input_name:
-                get_optimal_cap(bus_data[bus], dict_asset, bus, "input")
 
-    elif OUTPUT_BUS_NAME in dict_asset:
-        if not isinstance(output_name, list):
-            get_optimal_cap(bus_data[output_name], dict_asset, output_name, "output")
-        else:
-            for bus in output_name:
-                get_optimal_cap(bus_data[bus], dict_asset, bus, "output")
-    return
+def get_parameter_to_be_evaluated_from_oemof_results(asset_group, asset_label):
+    r"""
+    Determines the parameter that needs to be evaluated to determine an asset`s optimized flow and capacity.
+
+    Parameters
+    ----------
+    asset_group: str
+        Group that asset belongs to
+
+    asset_label: str
+        Label of the asset, needed for log message
+
+    Returns
+    -------
+    parameter_to_be_evaluated: str
+        Parameter that will be processed to get the dispatch and capacity of an asset
+
+    Notes
+    -----
+    Tested by:
+    - test_get_parameter_to_be_evaluated_from_oemof_results()
+    """
+    if asset_group in asset_groups_defined_by_influx:
```

Comment:
shouldn't `asset_groups_defined_by_influx` be in all CAPS ? I understand it it a global variable as I don't see it being defined within the function's scope

Reply:
Okay, was not sure about the convention. I thought only global variables defined in `constants*.py` could be in all caps. I decided not to put these into the constants itself as these are sensitive, more coding-related definitions. 

ad580d7d999120027c90b9dd8a3cb81e4a7c570c